TSOClient/tso.content/ContentSrc/Fonts/simdialogue.ini

```diff
@@ -3,4 +3,4 @@ path=Fonts/SimDialogueReBold.ttf
 [characters]
 ;See https://unicode-table.com/en for a nice overview of characters available in unicode
 ;Include space, and all standard printable ASCII characters and unicode arrows
-ranges=( ,~),( ,±),(´,¸),(º,»),(¿,ň),(Ŋ,ſ),(Ə,Ə),(Ơ,ơ),(Ư,ư),(Ș,ț),(Ȳ,ȳ),(ȷ,ȷ),(ə,ə),(ˆ,ˇ),(˘,˝),(̉,̉),(̒,̓),(̛,̛),(̣,̣),(̦,̦),(;,;),(΄,Ί),(Ό,Ό),(Ύ,Ρ),(Σ,ώ),(Ѐ,џ),(Ґ,ғ),(Җ,ҝ),(Ҡ,ң),(Ҫ,ҫ),(Ү,ҳ),(Ҷ,һ),(Ӏ,ӂ),(ӏ,ӑ),(Ӗ,ә),(Ӣ,ӣ),(Ө,ө),(Ӯ,ӯ),(Ӳ,ӳ),(Ạ,ỹ),(‐,―),(‘,‣),(‥,…),(‰,‰),(′,″),(‹,※),(€,€),(⃘,⃘),(℃,℃),(№,№),(℡,™),(Å,Å),(Ⅰ,ⅿ),(←,↕),(⇒,⇒),(⇔,⇔),(∀,∀),(∂,∃),(∇,∈),(∋,∋),(∑,∑),(√,√),(⇒,⇒),(⇔,⇔),(∀,∀),(∂,∃),(∇,∈),(∋,∋),(∑,∑),(√,√),(∝,∠),(∥,∥),(∧,∬),(∮,∮),(∴,∵),(∽,∽),(≈,≈),(≒,≒),(≠,≡),(≤,≧),(≪,≫),(⊂,⊃),(⊆,⊇),(⊥,⊥),(⊿,⊿),(⌒,⌒),(①,⑳),(─,┃),(┌,┌),(┏,┏),(┐,┐),(┓,└),(┗,┘),(┛,┝),(┠,┠),(┣,┥),(┨,┨),(┫,┬),(┯,┰),(┳,┴),(┷,┸),(┻,┼),(┿,┿),(╂,╂),(╋,╋),(■,□),(▲,△),(▼,▽),(◆,◇),(◊,○),(◎,●),(◯,◯),(★,☆),(♀,♀),(♂,♂),(♪,♪),(♭,♭),(♯,♯),(　,〕),(〝,〟)
+ranges=( ,~),( ,±),(´,¸),(º,»),(¿,ň),(Ŋ,ſ),(Ə,Ə),(Ơ,ơ),(Ư,ư),(Ǫ,ǫ),(Ș,ț),(Ȳ,ȳ),(ȷ,ȷ),(ə,ə),(ˆ,ˇ),(˘,˝),(̉,̉),(̒,̓),(̛,̛),(̣,̣),(̦,̦),(;,;),(΄,Ί),(Ό,Ό),(Ύ,Ρ),(Σ,ώ),(Ѐ,џ),(Ґ,ғ),(Җ,ҝ),(Ҡ,ң),(Ҫ,ҫ),(Ү,ҳ),(Ҷ,һ),(Ӏ,ӂ),(ӏ,ӑ),(Ӗ,ә),(Ӣ,ӣ),(Ө,ө),(Ӯ,ӯ),(Ӳ,ӳ),(ก,ฺ),(฿,๛),(ẞ,ẞ),(Ạ,ỹ),(‐,―),(‘,‣),(‥,…),(‰,‰),(′,″),(‹,※),(€,€),(⃘,⃘),(℃,℃),(№,№),(℡,™),(Å,Å),(Ⅰ,ⅿ),(←,↕),(⇒,⇒),(⇔,⇔),(∀,∀),(∂,∃),(∇,∈),(∋,∋),(∑,∑),(√,√),(⇒,⇒),(⇔,⇔),(∀,∀),(∂,∃),(∇,∈),(∋,∋),(∑,∑),(√,√),(∝,∠),(∥,∥),(∧,∬),(∮,∮),(∴,∵),(∽,∽),(≈,≈),(≒,≒),(≠,≡),(≤,≧),(≪,≫),(⊂,⊃),(⊆,⊇),(⊥,⊥),(⊿,⊿),(⌒,⌒),(①,⑳),(─,┃),(┌,┌),(┏,┏),(┐,┐),(┓,└),(┗,┘),(┛,┝),(┠,┠),(┣,┥),(┨,┨),(┫,┬),(┯,┰),(┳,┴),(┷,┸),(┻,┼),(┿,┿),(╂,╂),(╋,╋),(■,□),(▲,△),(▼,▽),(◆,◇),(◊,○),(◎,●),(◯,◯),(★,☆),(♀,♀),(♂,♂),(♪,♪),(♭,♭),(♯,♯),(　,〕),(〝,〟)
```

Comment:
There seems to be a problem with your addition for the thai character range - you might need to use a code editor that better supports UTF-8 encoding.

The end character is appearing as broken (some editors combine it with the comma), it should probably be:
๛

Reply:
The end character isn't broken, it's a combining character. Just like characters here: `(̉,̉),(̒,̓),(̛,̛),(̣,̣),(̦,̦)`. By the way, the whole Thai range is `(ก,ฺ),(฿,๛)`. I've split it because U+0E3B to U+0E3E are empty characters.

Reply:
It was failing to parse, which is broken. All I was getting out of it were the english characters

Reply:
As a quick preview, you can see all the characters that were added to the font in `ContentSrc/test.png` when compiling.

Reply:
It is not the SimDialogue font so there are many Xs… I don't know which characters are missing.

Reply:
I'm confused. I can't compile the font myself, I'm not smart enough not to get crazy while trying. Have you tried to replace `(ก,ฺ),(฿,๛)` with `(ก,๛)`?

Reply:
Ah, looks like it was an issue with visual studio not properly copying assets that are changed. It's good in that case.